### PR TITLE
chrome dino game automation in python

### LINKED ISSUE #713: Automation of Google Chrome Dino Game
I will be using python and its libraries such as pyautogui  and pillow to automate chrome dinosaur game with NLP to teach dino about the hurdles coming in between.

### PR DESCRIPTION
**Pull request by oshi36**
# Description

I had used python to automate chrome dinosaur game using pyautogui and pillow .

Fixes #713

## Type of change
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by HarshCasper** [comment]
Few Changes before the PR is reviewed: 

- Add a README to document your Code/Approach 
- Add Screenshots/GIF to show that your Script is working 
- Make apt changes on the Python README 
- Put Comments/Docstrings on the Code

**Comment by oshi36** [comment]
@HarshCasper  I had done the above mentioned changes .

File: Python/chromedino_script/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Chrome Dino Game Automation Using Python
+
+For doing automation I had used python libraries such as pyautogui and pillow .
+
+To play dino game we need to take care of two obstacles : first the cactus coming in between and second the bird flying above.
+
+To know the location of the obstacles first we need to find the X and Y locations of them on screen . Check coordinates.py file
```
Comment (HarshCasper) [inline Python/chromedino_script/README.md:7]: Can you make this more relevant? 

Usually follow this pattern while documenting your Code: 

- What does your Script does? 
- The files in them and their individual roles (Don't ask the user to check coordinates.py file) 
- Requirements to run these Scripts (Place them in a `requirements.txt` file

File: Python/chromedino_script/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+So , we have to match the gray pixels and if obstacle  is close to dino then  make a jump. For this take a image with obstacles coming in between using ImageGrab from pillow library and then find out the pixels in which the obstacles are coming. Check enemy_location_checker.py file
+
+# To run bot
+First increase the search width for evey second the bot goes (Use time module from python) to simulate the dino acceleration.
+Second get the background color (night and day) .
+Third reverse loop for dino to get searched from back of screenshot to identify obstacle and it will speed up the program.
+
+Check main.py file
+
+After running the main.py file go to chrome browser where dino game is opened(chrome://dino) and then start game by pressing space bar and it will play automatically.
```
Comment (HarshCasper) [inline Python/chromedino_script/README.md:21]: Can you add proper step-wise instructions to run the scripts?

File: Python/chromedino_script/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+For doing automation I had used python libraries such as pyautogui and pillow .
+
+To play dino game we need to take care of two obstacles : first the cactus coming in between and second the bird flying above.
+
+To know the location of the obstacles first we need to find the X and Y locations of them on screen . Check coordinates.py file
+
+# To identify enemies
+Then to make dino identify the cactus and bird , captured the image of two make a frame of it and trained dino to escape it.
+Luckily , we have only three colors in this game : black(for night) , white(for day)  and gray.(for obstacles and dino)
+So , we have to match the gray pixels and if obstacle  is close to dino then  make a jump. For this take a image with obstacles coming in between using ImageGrab from pillow library and then find out the pixels in which the obstacles are coming. Check enemy_location_checker.py file
```
Comment (HarshCasper) [inline Python/chromedino_script/README.md:12]: Use a Checker like Grammarly to verify if the content is consistent. Highlight the file names

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Please take a look at some other docs for various scripts to see on how you can format your doc. The Code look good to me 👍

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
@all-contributors please add @oshi36 for code

**Comment by HarshCasper** [comment]
@all-contributors please add @oshi36 for documentation

**Pull request merged**